demo/src/main/java/com/novoda/storagepathfinder/demo/LandingActivity.java

```diff
@@ -24,20 +25,20 @@ protected void onCreate(Bundle savedInstanceState) {
         assetCloner = DemoDependenciesFactory.createAssetCloner(getApplicationContext());
 
         AndroidDeviceStorageInspector storageInspector = DemoDependenciesFactory.createStorageInspector(getApplicationContext());
-        List<DeviceStorageRoot> deviceStorageRoots = storageInspector.getDeviceStorageRoots();
+        List<StoragePath> deviceStoragePaths = Collections.emptyList(); // TODO use the inspector :D
```

Comment:
This needs to be fixed. We need new logic for what the demo wants to do, as before it was just a list of paths, now there is a choice of which paths we want